package-lock.json

```diff
@@ -5,10 +5,12 @@
     "requires": true,
     "packages": {
         "": {
+            "name": "nisaba",
```

Comment:
I'm surprised to see this here :thinking:  Or maybe the "package" was already here, but without the name?

Reply:
It was but without the `name` key -  the dependencies were listed, for example. It's the first package under `packages`, with an empty key... 🤷

Reply:
After running `npm install` locally, this line disappeared :laughing: